### ISSUE TITLE
WebUI does not show binaries of 'released products'

### ISSUE THREAD

**Issue by DimStar77** [openSUSE/open-build-service#4802]
The product, built in 000product in openSUSE:Factory, is 'released' to openSUSE:Factory:ToTest

Seems the webui does not show the binaries when requesting to see them.

e.g., using osc:

```
> osc ls -b openSUSE:Factory:ToTest/000product:openSUSE-Tumbleweed-Kubic-dvd5-dvd-x86_64/images
images/local
 _channel
 _statistics
 openSUSE-Tumbleweed-Kubic-DVD-x86_64-Snapshot20180403-Media.iso
 openSUSE-Tumbleweed-Kubic-DVD-x86_64-Snapshot20180403-Media.iso.sha256
 openSUSE-Tumbleweed-Kubic-DVD-x86_64-Snapshot20180403-Media.report
images/i586
images/x86_64
```

The webui shows it like this:

![screenshot from 2018-04-05 09-40-35](https://user-images.githubusercontent.com/532559/38353015-70f03654-38b5-11e8-85d8-30ee3bc0ea83.png)

Note that the 'header' shows only 000product, not the sub-item I actually requested (this might be a generic multibuild-related issue, did not (yet) check other multibuild packages)

**Comment by coolo** [comment]
this has nothing to do with released or not - the webui is just broken for multibuild packages. Ever checked installation-images binaries?

**Comment by coolo** [comment]
And it's *super* annoying

**Comment by hennevogel** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Fix multibuild binary view](https://trello.com/c/Ihi8rRMj/1384-fix-multibuild-binary-view)

**Comment by adrianschroeter** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Fix multibuild binary view](https://trello.com/c/RgM3GYNC/1407-fix-multibuild-binary-view)

**Comment by DavidKang** [comment]
Fixed in PR #4924
